sdk/src/main/scala/com/horizen/account/state/AccountStateView.scala

```diff
@@ -25,19 +36,27 @@ import scala.collection.JavaConverters.collectionAsScalaIterableConverter
 import scala.util.Try
 
 class AccountStateView(
-  metadataStorageView: AccountStateMetadataStorageView,
-  stateDb: StateDB,
-  messageProcessors: Seq[MessageProcessor])
-  extends StateView[SidechainTypes#SCAT]
+    metadataStorageView: AccountStateMetadataStorageView,
+    stateDb: StateDB,
+    messageProcessors: Seq[MessageProcessor]
+) extends StateView[SidechainTypes#SCAT]
```

Comment:
This is a file reformat, but not with the Scalafmt settings, if I'm not mistaken. If we do reformatting then with common settings please.

Reply:
After your comment about Scalafmt, I changed the formatter in Intellij in order to use Scalafmt but apparently something went wrong with this file.